### DISCUSSION TITLE
Linelists

### DISCUSSION THREAD

**Discussion by nimishak-cosmos** [spacetelescope/jdaviz#1519] [General]
Hi,

Quick comments for the nebular line lists used here 
(1) Common_nebular.csv: this list does not have any digits after decimal, which are quite important when we determine redshifts, systemic velocities etc, and the vacuum and air wavelengths differ by decimal places too. So, it is important to update that list.
(2) SDSS-IV.csv has wavelengths in air and SDSS.csv has wavelengths in vacuum. Depending on what people use, people get different (erroneous) science results, so it would be good to note them too, somewhere.
(3) Common_stellar.csv: assuming that the linelist has wavelengths in vacuum (as I infer from Lya wavelength), a quick look at few other lines show that they are not correct (e.g. CIII], Mg II etc).
 
Thanks & Best Regards,
Nimisha

**Comment by pllim** [comment]
Thank you. These comments are being tracked in the following GitHub issues:

* https://github.com/spacetelescope/jdaviz/issues/1517
* https://github.com/spacetelescope/jdaviz/issues/1521
